docs/installation/transfer-installation.md

```diff
@@ -0,0 +1,79 @@
+---
+id: transfer-installation
+sidebar_label: Transfer your installation
+title: Transfer your installation
+description: A guide on how to transfer your autobrr installation to another server
+keywords:
+  [
+    autobrr,
+    installation,
+    setup,
+    seedbox,
+    shared seedbox,
+    swizzin,
+    swizzin.net,
+    sbio,
+    seedbox.io,
+    ultra,
+    ultra.cc,
+    seedhost.eu,
+    seedhost,
+    saltbox, 
+    quickbox,
+  ]
+pagination_prev: introduction
+pagination_next: configuration/indexers
+---
+
+# Transfer your installation
+
+This guide will walk you through the steps needed to transfer your settings such as filters, indexers, IRC networks, feeds, clients, notifications and API keys of your current autobrr installation from one host to another.
+
+Settings like hostname / IP, port, baseUrl, logPath or logLevel are not transferred and have to be adjusted again in `config.toml` after the transfer in case you made any changes to this file.
```

Comment:
Can this be emphasized in anyway? We've had a few people forget this.

:::info
Settings like hostname / IP, port, baseUrl, logPath or logLevel are not transferred and have to be adjusted again in `config.toml` after the transfer in case you made any changes to this file.
:::

Reply:
Done.